uniconnect-epfl/uniconnect
# feat: improved layout and functionality of registration flow

# What I did
- Cleaned the layout throughout the registration flow, made it consistent (background color, input field size).
- Made the layout not move when the keyboard goes up.
- Made some fields required (like first name for example), you can't continue without filling it. It will display a toast to warn you.
- Made it so the labels are persistent when moving back and coming back to the interest screen.

# How I did it
- Modified already existing screens and styles files.

# How to verify it
Launch the app in expo go, create a new account, check the layout and try to break the functionalities, skip steps and so on...

# Demo video

https://github.com/uniconnect-epfl/uniconnect/assets/93329823/1b3cef35-0d2a-4af6-9a1c-8c93f0978fde

# Pre-merge checklist
The changes I have introduced:
